discovery-provider/plugins/notifications/src/__tests__/mappings/create.test.ts

```diff
@@ -81,9 +81,9 @@ describe('Create Notification', () => {
         data: {
           id: 'timestamp:1589373217:group_id:create:track:user_id:1',
           type: 'UserSubscription',
-          entityId: 10,
-          entityOwnerId: 1,
-          entityType: 'track'
+          entityIds: [10],
```

Comment:
this was a fix for a bug that happened over the weekend where opening the create push notif crashed the app because of mobile deep linking expecting a differently named attribute